helevents/service_tasks/daily_tasks.sh

```diff
@@ -25,6 +25,11 @@ if [ $? -ne 0 ]; then
     echo "PKS osoiteluettelo update signaled failure"
 fi
 
+timeout --preserve-status -s INT 60m python manage.py anonymize_past_signups
+if [ $? -ne 0 ]; then
+    echo "Past signups anonymization failure"
+fi
+
```

Comment:
Actually this whole script [has been moved to the Azure repo](https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines/pullrequest/6825), so this change needs to be added there after this PR is merged. I guess it doesn't hurt to have it in this file as well especially now that it's already added, but unless some issues come up, we are removing this whole file soon 🙂

Reply:
Oh yes, you told in some daily that you guys are moving cron jobs to Azure. I will create a separate PR there after this PR is merged.